### PR TITLE
Fix MixingConfing w/ Lookups integration

### PR DESCRIPTION
**Pull request by CPerezz**
This fixes all the errors in the Mixing config and also introduces the base conversion inside of it.

### CODE REVIEW

File: keccak256/src/circuit.rs [hunk 1]
```diff
@@ -242,9 +239,22 @@ impl<F: FieldExt> KeccakFConfig<F> {
...
-            self.mixing_config.assign_state(region, offset, state, out_state, flag, next_mixing, round, round)?;
+            self.mixing_config.assign_state(
+                region,
+                offset,
+                state,
+                out_state,
+                flag,
+                next_mixing,
+                round,
+                round,
```
Comment (ChihChengLiang) [inline keccak256/src/circuit.rs:256]: Why do we have two args of rounds here? This should always be the same round.

**Review by ChihChengLiang — APPROVED** [review]
LGTM.
I added a question, but it's not urgent to fix in this PR.

File: keccak256/src/keccak_arith.rs [hunk 2]
```diff
@@ -106,6 +106,7 @@ impl KeccakFArith {
     ) -> StateBigInt {
         if let Some(next_input) = next_input {
             let out_1 = KeccakFArith::absorb(a, next_input);
+            // TODO: Base conversion missing
```
Comment (therealyingtong) [inline keccak256/src/keccak_arith.rs:109]: Will this be addressed in another PR?

**Review by therealyingtong — APPROVED** [review]
LGTM, with one question

File: keccak256/src/keccak_arith.rs (hunk 2 above)
Comment (CPerezz) [reply keccak256/src/keccak_arith.rs:109]: That's a leftover.. Sorry.
Removed and merging!
